Readme.md

```diff
@@ -86,33 +116,41 @@ paths:
                 type: string
                 example: "2023-02-06T09:15:45Z"
 ```
-The reference prices are updated every 10 minutes until the beginning of the delivery hour.
+
+The reference prices for each delivery hour are updated every 10 minutes until gate closure. The updates are sometimes published with some delay, usually something between 0 and 30 seconds but without any clear patterns.
+
+Once the price is available for one delivery hour, it's available for the rest of the tradable hours (That is until the end of the same day).
+
+There is a limit on the number of requests we can submit to the API, about 50 per minute and 1000 per hour.
 
 ## Reporting:
 
 The reporting frontend displays the following dashboard:
 ![img.png](dashboard.png)
-It can be accessed by the traders from their browsers.
-The dashboard is updated with all the current values every 5 Minutes. 
+
+It can be accessed by the traders from their browsers. Since this is the traders' main source of validation, they might get impatient around the time the reference price is supposed to be published and start massively refreshing their reporting dashboard.
```

Comment:
Maybe this is a little bit too close to the truth. 😆  Maybe this frighten candidates. 😅

Reply:
I wanted to have something about how the users use the system and this refers to a couple of important topics like elasticity VS scalability and managing costs in the cloud.

I would like to try it and see what happens!